docker/include/network/scripts/start_networking

```diff
@@ -31,10 +31,12 @@ subnet=${IP_SUBNET:-$random_subnet}
 echo Configuring with subnet 10.20.$subnet.XX
 echo dhcp-range=10.20.$subnet.100,10.20.$subnet.254 >> /etc/dnsmasq.conf
 
-# Use ourself as the DNS and NTP server explicitly specified
+# Use ourself as the DNS server explicitly specified
 ip addr add 10.20.$subnet.2 dev $LOCAL_IF
 echo dhcp-option=6,10.20.$subnet.2 >> /etc/dnsmasq.conf
-echo dhcp-option=42,10.20.$subnet.2 >> /etc/dnsmasq.conf
```

Comment:
Should we delete this? Leaving this in place would allow the DUT to either learn NTP from DHCP option, or have it statically configured, so we'll cover both options?

Reply:
Yes, on the issue  #845 there's been additional comments from Trevor, as well as your comment from last week on having both test cases, so this PR will be updated accordingly